ipol_demo/modules/core/core.py

```diff
@@ -768,24 +768,21 @@ def copy_blobset_from_physical_location(self, demo_id, work_dir, blobset_id):
         returns:
         """
         try:
-            demo_blobs, _ = self.post(
-                "api/blobs/get_blobs", "post", data={"demo_id": demo_id}
-            )
-            demo_blobs = demo_blobs.json()
+            demo_blobs, status = self.post(f"api/blobs/demo_blobs/{demo_id}", "get")
         except json.JSONDecodeError as e:
             self.logger.exception(
-                f"Blobs get_blobs didn't return valid json at Core's copy_blobset_from_physical_location. {e}"
+                f"Blobs didn't return valid json at Core's copy_blobset_from_physical_location. {e}"
             )
             raise IPOLCopyBlobsError("Couldn't reach blobs")
 
-        if not demo_blobs["sets"] or demo_blobs["status"] == "KO":
+        if not demo_blobs or status != 200:
             self.logger.exception(
-                "Blobs get_blobs returned KO at Core's copy_blobset_from_physical_location"
+                "Failed to get blobs at Core's copy_blobset_from_physical_location"
             )
             raise IPOLCopyBlobsError("Couldn't reach blobs")
```

Comment:
Let's put the text of the exception in a variable and use it to log and the raise it in the next line.

Reply:
Can't use exception info outside of an exception

Reply:
What I mean is that in l.783 we have a complete message and right after we raise an exception with another message. I propose to do something like
            error_msg = "Failed to get blobs at Core's copy_blobset_from_physical_location"
            self.logger.exception(error_msg)
            raise IPOLCopyBlobsError(error_msg)